docs/pieces/apps/trello.mdx

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
Please move this into piece folder, the docs will be generated from it.

Check discord example: https://github.com/activepieces/activepieces/blob/main/packages/pieces/apps/src/lib/discord/discord.mdx